main.md

```diff
@@ -193,180 +199,327 @@ Note: Applications MUST ensure that their authorization data types do not collid
 The following example shows how an implementation could utilize the namespace `https://scheme.example.org/` to ensure collision resistant element names.
 
 ```JSON
-{ 
-   "type":"https://scheme.example.org/files",
-   "locations":[ 
+{
+   "type": "https://scheme.example.org/files",
+   "locations": [
       "https://example.com/files"
    ],
-   "permissions":[ 
-      { 
-         "path":"/myfiles/A",
-         "access":[ 
+   "permissions": [
+      {
+         "path": "/myfiles/A",
+         "access": [
             "read"
          ]
       },
-      { 
-         "path":"/myfiles/A/X",
-         "access":[ 
+      {
+         "path": "/myfiles/A/X",
+         "access": [
             "read",
             "write"
          ]
       }
    ]
 }
 ```
+## Relationship to "scope" parameter {#scope}
+
+`authorization_details` and `scope` can be used in the same authorization request for carrying independent authorization requirements. 
+
+The AS MUST keep both requirements sets separate when reflecting this data to the client and passing this data through to resource servers.
+
+For user convenience, the AS is supposed to merge the requirements when asking the user for consent.
+
+### Scope value "openid" and "claims" parameter
+
+OpenID Connect [@OIDC] specifies the JSON-based `claims` request parameter that can be used to specify the claims a client (acting as OpenID Connect Relying Party) wishes to receive in a fine-grained and privacy preserving way as well as assign those claims to a certain delivery mechanisms, i.e. ID Token or userinfo response. 
+
+The combination of the scope value `openid` and the additional parameter `claims` can be used beside `authorization_details` in the same way as every other scope value and, potentially, further parameter providing additional data for the respective scope to the authorization process. 
```

Comment:
I get what you're saying but this is awkward to add here because it's not defined in OIDC.

```suggestion
The combination of the scope value `openid` and the additional parameter `claims` can be used beside `authorization_details` in the same way as every other non-OIDC scope value. 
```